youtube_dl/extractor/kuwo.py

```diff
@@ -78,7 +79,7 @@ def _real_extract(self, url):
             errnote='Unable to get song detail info')
 
         song_name = self._html_search_regex(
-            r'<h1[^>]+title="([^"]+)">', webpage, 'song name')
+            r'(?s)class="(?:[^" ]+ +)*title(?: +[^" ]+)*".*?<h1[^>]+title="([^"]+)"', webpage, 'song name')
```

Comment:
According to the [W3C HTML5 syntax spec](http://www.w3.org/TR/html5/syntax.html), using ```'\s'``` to match whitespaces is better here.